src/opentimelineio/stackAlgorithm.cpp

```diff
@@ -144,7 +146,7 @@ _normalize_tracks_lengths(
         }
     }
 
-    for (int i = 0; i < tracks.size(); i++)
+    for (std::size_t i = 0; i < tracks.size(); i++)
```

Comment:
tracks.size() returns a size_t; if you just use size_t, you don't need to include the c header file.